markdown/api-docs/catalog/products-overview.md

```diff
@@ -540,6 +541,21 @@ lineNumbers: true
 * Must have a SKU code (unless they’re a base variant)
 * In the case of non-base variants, variants will relate to a particular combination of variant option values - such as “small” and “red”
 
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--warning">
+<div class="HubBlock-content">
+    
+<!-- theme: warning -->
+
+### V2 SKU rules will override Variant pricing
+> Creating SKU rules via the V2 API or via CSV import will override any Variant sale pricing added to a product via the Control Panel, V3 API or Price Lists UI.

```

Comment:
`... will alter or override any Variant price or sale price added to a product ...  `

I say this because if the rule is a Percent or relative adjuster, it'll be applied on top of the variant price.  But if it's an absolute rule it'll override the price.

Reply:
@mikrdgz do you mind addressing this?

Reply:
Done